docs/language-spec.md

```diff
@@ -1664,15 +1664,16 @@ message MyMessage {
 
 :::info
 
-Groups cannot be used in files that use the proto3 syntax.
+Groups cannot be used in files that use the proto3 syntax. Field Cardinality is 
+required in the proto2 syntax.
```

Comment:
I think this was fine the way it was before. When we were talking Friday and looking at this, I was expecting a note along these lines because there is one like this for normal fields.

But now that you've fixed the grammar production below, this is not necessary. The grammar change alone adequately communicates this fact IMO.
```suggestion
Groups cannot be used in files that use the proto3 syntax.
```